### ISSUE TITLE
Integration Guide - General Rules 

### ISSUE THREAD

**Issue by app-generator** [app-generator/jinja-berry-dashboard#1]
Hello guys, 

As a general rule, the goal is to reuse the Django work with minimal effort WITHOUT adding anything functional (DB, login, logout, registration). Here, only the pages should be served without issues. 

this is the one and only evolution required. 

Here is the migration/integration guide:

- assets should remain untouched 
- all pages preserved (no deletion)
- assets loaded via the `static` tag
  - only adapted for flask
  - No more hardcoded `/static/assets` usage
- No dependencies added
- No database

Basically, all pages should be updated to be rendered by Flask without any functional code for:
- sign in
- sign up
- reset pass
+ other actions

Only the pages should be linked 

signIN <-> registration <-> reset password and other linked
Logout link for instance is linked to signIN in order to simulate the action

Let me know here or via UpW if something is now clear.
